20240329_CHWIDE_component_separated/README.md

```diff
@@ -0,0 +1,28 @@
+# Documentation for CHWIDE component separated maps
+
+#### Data description
+We provide foreground cleaned CMB maps obtained with NILC for CMB-S4 Chile wide survey. There are separate maps for E and B-mode polarizations. 
+Currently, about 25 realizations are provided for testing purposes. The masks used for the E and B maps are also provided. All maps are at `NSIDE=2048` 
+and `lmax=4096`, with 2.1 arcmin gaussian beam. The current maps are only with the PySM medium complexity foreground model.
```

Comment:
do you have any detail about the software (repository, version) used for this? or link to previous paper?

Reply:
The code is not on github yet. Creating a repo for the massively parallel NILC implementation to the level required for others to access it, is beyond the scope of this documentation PR. I agree that having these information would be good. I added a link to the slides I showed on the telecon, which have some additional details on the pipeline.

Reply:
sure, can you write "NILC" implementation not released yet? So people know that is a private code

Reply:
Added.